scr/hw6/ArrayDataException.java

```diff
@@ -0,0 +1,13 @@
+package hw6;
+
+public class ArrayDataException extends Throwable implements MyException {
```

Comment:
Typical creating own exceptions - extends `RuntimeExceptions`. Hardly ever - extends `Exception`. `Throwable` includes also `Error` for example. Every exception is throwable, but opposit way is incorrect